docs/changehistory/NextVersion.md

```diff
@@ -50,3 +51,29 @@ The Xml and JSON representations of a custom attribute (and related TypeScript i
 ## Improve the performance of the ECSchemaRpcLocater
 
 Improve the performance of the ECSchemaRpcLocater by making all of the underlying ECSchemaRpcInterface methods GET by default so responses are cached by default. Previously each client had to set the methods to be GET or they would default to POST and were not cached.
+
+## Mathematical equations formatting
+
+The formatter now supports parsing mathematical equations. Ex :
+```Typescript
```

Comment:
where is the actual ts code for this? what does it look like?

Reply:
it's in the tests. 
It looks like this : 
```
// Async version 
const quantityProps = await Parser.parseIntoQuantity("2 FT 6 in + 9", format, unitsProvider);

// Sync version
const parseResult = Parser.parseToQuantityValue("2 FT 6 in + 9", formatU, meterConversionSpecs);
```

Reply:
consumers will not be looking at the tests, theyll look at the the changelog here, and this should probably have a ts snippet of the change

Reply:
Users will use the quantity formatter with the boolean flag. 
They will not call these directly. Should I add an example anyway ?